### ISSUE TITLE
Show status of all brokers

### ISSUE THREAD

**Issue by edenhill** [edenhill/kafkacat#25]
E.g., if they are up

**Comment by fsaintjacques** [comment]
Would it make more sense to add information in the metadata request instead?

**Comment by edenhill** [comment]
Yep, probably a good idea.

**Comment by fsaintjacques** [comment]
on my fork I changed the metadata query to output json instead.

```
{
  "brokers": [
    { "id":54, "host":"broker54:9092" },
    { "id":53, "host":"broker53:9092" }
  ],
```

It would be trivial to add a field "status" in the broker object.

**Comment by edenhill** [comment]
Yep, thats awesome, but should be enabled with an option (e.g., `-j`)

**Comment by edenhill** [comment]
Enabling and exposing the librdkafka stats would solve this issue as well, since stats show detailed state for each broker.

**Comment by fsaintjacques** [comment]
:+1:
